src/main/java/com/xceptance/xlt/mastercontroller/Main.java

```diff
@@ -183,7 +183,7 @@ private static void setupHttpsProxy(final MasterControllerConfiguration config)
      *            master controller configuration
      * @return the HessianProxyFactory
      */
-    private static HessianProxyFactory getHessianProxyFactory(final MasterControllerConfiguration config)
+    private HessianProxyFactory getHessianProxyFactory(final MasterControllerConfiguration config)
```

Comment:
Shouldn't this be protected to allow overrides?

Reply:
Done.